src/pages/hcp/api-docs/webhook/[[...page]].tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// Lib
+import { fetchCloudApiVersionData } from 'lib/api-docs/fetch-cloud-api-version-data'
+// View
+import OpenApiDocsView from 'views/open-api-docs-view'
+import {
+	getStaticPaths,
+	getStaticProps as getOpenApiDocsStaticProps,
+} from 'views/open-api-docs-view/server'
+// Types
+import type { GetStaticProps, GetStaticPropsContext } from 'next'
+import type { OpenAPIV3 } from 'openapi-types'
+import type {
+	OpenApiDocsParams,
+	OpenApiDocsViewProps,
+	OpenApiDocsPageConfig,
+} from 'views/open-api-docs-view/types'
+
+/**
+ * Configure the specific spec file we want to fetch,
+ * as well as some other minor content details and bells & whistles.
+ */
+const PAGE_CONFIG: OpenApiDocsPageConfig = {
+	basePath: '/hcp/api-docs/webhook',
+	productSlug: 'hcp',
+	githubSourceDirectory: {
+		owner: 'hashicorp',
+		repo: 'hcp-specs',
+		path: 'specs/cloud-webhook',
+		ref: 'main',
+	},
+	statusIndicatorConfig: {
+		pageUrl: 'https://status.hashicorp.com',
+		endpointUrl:
+			'https://status.hashicorp.com/api/v2/components/0q55nwmxngkc.json',
+	},
+	navResourceItems: [
+		{
```

Comment:
For consistency we should add the tutorials library and support links here. 

```
		{
			title: 'Tutorial Library',
			href: '/tutorials/library?edition=hcp',
		},
		{
			title: 'Community',
			href: 'https://discuss.hashicorp.com/',
		},
```

Reply:
We still don't have tutorials for webhooks so I will only add the community